applicationinsights-common.ts

```diff
@@ -0,0 +1,5 @@
+export { Util, CorrelationIdHelper, UrlHelper } from './AppInsightsCommon/Util';
```

Comment:
I would recommend you move all files inside applicationinsights common down into a sub dir. That way this file can be part of the root AppInsightsCommon directory.